lib/gooddata/lcm/helpers/check_helper.rb

```diff
@@ -15,7 +15,7 @@ def check_params(specification, params)
             value = params.send(param_name)
             type = specification[param_name][:type]
             if value.nil? || (value.is_a?(String) && value.empty?)
-              if specification[param_name][:opts][:default]
+              if !specification[param_name][:opts][:default].nil?
```

Comment:
Refers to 'specification' more than self (maybe move it to another class?) (FeatureEnvy)